examples/edit_custom_iocs/crowdstrike-ioc.go

```diff
@@ -19,16 +19,29 @@ import (
 
 // getCrowdstrikeIOCs returns a list of all the Custom IOC values in the system
 func getCrowdstrikeIOCs(client *client.CrowdStrikeAPISpecification) ([]string, error) {
-	params := ioc.NewIndicatorCombinedV1Params().WithDefaults()
-	res2, err := client.Ioc.IndicatorCombinedV1(params)
-	if err != nil {
-		// fmt.Println(res.Error())
-		return []string{}, err
-	}
 	iocs := []string{}
-	for _, ioc := range res2.GetPayload().Resources {
-		iocs = append(iocs, ioc.Value)
+	var limit, offset, total int64
+	limit = 2000
+	offset = 0
+	total = 0
+
+	for offset >= total {
```

Comment:
There seems to be a problem in the pagination logic.

I have environment with only one IOC, but this loop never ends as the offset is always greater than the total.

Reply:
There is already one example that deals with pagination. (You don't need to copy that solution, just mentioning it for the reference).

https://github.com/CrowdStrike/gofalcon/blob/main/examples/falcon_cleanup_pods/main.go#L136-L161 

In this example there is go channel established and asynchronous go routine fills in the channel with the data from API. That way the caller doesn't need to wait for all the items to be fetched from the api can can process the items as they appear in the channel. That is theoretically the ultimate solution for the pagination, but your approach is fine as well, really mentioning this just for reference.

Reply:
The following patch fixes pagination for me.
```
-	for offset >= total {
+	for offset <= total {
```